blocks/extra-large-manual-promo-block/themes/news.json

```diff
@@ -32,5 +38,29 @@
 				}
 			}
 		}
+	},
+	"xl-manual-promo-overline-hover": {
+		"styles": {
+			"default": {
+				"color": "var(--text-color-subtle)"
+			},
+			"desktop": {}
+		}
+	},
+	"xl-manual-promo-headline-hover": {
+		"styles": {
+			"default": {
+				"color": "var(--text-color-subtle)"
+			},
+			"desktop": {}
+		}
+	},
+	"xl-manual-promo-img-with-link": {
+		"styles": {
+			"default": {
+				"opacity": 0.8
```

Comment:
This should work as 
```
"components": {
  "link-hover": {
      "opacity": 0.8
    }
}
```

Reply:
@vgalatro Instead of link-hover, it's working with image